test/dynamicppl/loglikelihoods.jl

```diff
@@ -0,0 +1,48 @@
+module DynamicPPLLogLikelihoodsTest
```

Comment:
I believe this can actually be moved to DynamicPPL.jl's test suite proper under the `test/ext` dir, by just replacing the specific sampler calls with explicit constructions of `Chains`:)

THis is from before we had DynamicPPLMCMCChainsExt.

Reply:
Okay, will delete from here and move to MCMCChainsExt in the followup PR

Reply:
Actually, aren't these tested here already? https://github.com/TuringLang/DynamicPPL.jl/blob/master/test/pointwise_logdensities.jl

I think this is ok to just delete.

Reply:
Oh yeah, good catch 👍